### DISCUSSION TITLE
Creating Extensions

### DISCUSSION THREAD

**Discussion by Pixandi** [Minestom/Minestom#2119] [Help] [answered]
I'm new to Minestom. Can some1 explain how I can make my own Extensions. Because the wiki is outdated.

**Comment by Protonull** [comment, marked as answer]
Minestom no longer natively supports extensions. You'd need to have a server that shades https://github.com/hollow-cube/minestom-ce-extensions and invokes the appropriate API. After that, you should be able to use most readymade extensions. If you're making a new extension, then you'd need to compileOnly depend on https://github.com/hollow-cube/minestom-ce-extensions.

**Reply by zax71** [reply]
Currently, Gradle subprojects/libs are what is recommended instead of extensions
